lib/pages/profile_detail/profile_detail_state.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/foundation.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:morning_weakers/models/user/user.dart';
+
+part 'profile_detail_state.freezed.dart';
+
+@freezed
+abstract class ProfileDetailState with _$ProfileDetailState {
+  const factory ProfileDetailState({
+    @Default(
```

Comment:
```suggestion
    @Default(dummyUser())
```
がたしかconstantじゃないからできひんのやったっけな確か...
@Defaultを書かずに@requireにしてインスタンス化するときに必ずdummyUser()を渡すようにすれば良いかな？？

Reply:
Stateクラスのインスタンス化時に初期値渡してもだめなんよねーー

Reply:
`ProfileDetailController() : super(ProfileDetailState(user: dummyUser()));`
Stateからconst外せば一応いけるっぽい！
ここconstつけるのとつけないのでどう変わるかわかってないけど😇

Reply:
いや、それ自体コンパイルはできるんやけど、Pageを開いたら、Userクラスの中身が@requiredになっとるのに値がないって怒られるんよーー

Reply:
おお？lintというかIDEで怒られる感じ？
それともrunで？
podsがだめでこのbranch runできてないw

Reply:
えええ？wPod何もpushしてないのにw
runで！

Reply:
/ios/でpod installしたらいけるかも！

Reply:
debb5d3
ここで差分でてない？
pod installしてみる！

Reply:
ごめん直した！